package.json

```diff
@@ -29,6 +29,7 @@
     "installer": "node ./script/installer.js"
   },
   "devDependencies": {
+    "auto-launch": "^2.0.1",
```

Comment:
auto-launch is a dependency for the actual app. So please move into `dependencies` of `src/package.json`.